jsArrays/task3.js

```diff
@@ -0,0 +1,7 @@
+const numbers = [10, 20, 30, 40, 50];
+
+const sum = numbers.reduce((accumulator, currentValue) => {
```

Comment:
Можна і так, але буде легше:
```suggestion
const sum = numbers.reduce((accumulator, currentValue) => accumulator + currentValue)
```